app/qml/FeaturePanel.qml

```diff
@@ -202,7 +202,7 @@ Drawer {
                 }
 
                 property QtObject constraint: QtObject {
-                  property color validColor: "black"
+                  property color validColor: InputStyle.panelBackgroundDarker
```

Comment:
Hmm... panelBackgroundDarker is defined as `#575757` in InputStyle, but the design has `#999999` from what my color picker says :-)  I think we should introduce another color in the style for this.